src/build_post.sh

```diff
@@ -11,11 +11,6 @@ else
   export MOD_PATH=${cwd}/lib/modulefiles
 fi
 
-# Check final exec folder exists
-if [ ! -d "../exec" ]; then
-  mkdir ../exec
-fi
-
```

Comment:
Shouldn't there still be a check for the ../bin directory?

Reply:
Oh, it looks like this is now in the install_all.sh script?

Reply:
It's not even in the install_all.sh script.  I assume that users are always going to run the top-level script build_all.sh, not the ones that it in turn calls (e.g. build_post.sh).  They can control which codes are build via the configuration file regional_build.cfg.

Reply:
Yeah, I'm seeing that now.  Wasn't there a way previously that users could call the individual scripts in a stand-alone fashion?  I thought there was, but if we're going to require users to modify regional_build.cfg only, then run build_all.sh, I'm OK with that option.  I guess that should be explicitly mentioned in the documentation.

Reply:
Well all these scripts are not even going to be in the release, so we don't need to document it.  I guess they're eventually going to removed from develop as well, so there's even less incentive to document!  I could put the directory check/create stanzas in the sub-scripts, but that requires retesting, which may not be worth it (since the configuration file already provides a mechanism for limiting what you want to build).

Reply:
Nope, it's fine as is.  Approving now.

Reply:
Thanks, merging now.